### PR TITLE
Remove support for first version of composer

### PR DESCRIPTION
**Pull request by devanych**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
     "require": {
         "php": "^7.4|^8.0",
         "ext-json": "*",
-        "composer-plugin-api": "^1.0|^2.0",
+        "composer-plugin-api": "^2.0",
         "jfcherng/php-sequence-matcher": "^3.2",
         "yiisoft/var-dumper": "^1.1"
     },
     "require-dev": {
+        "composer/composer": "^2.0",
```
Comment (vjik) [inline composer.json:27]: Why require `composer/composer` ?

Comment (devanych) [reply composer.json:27]: For tests (`require-dev`).

Comment (devanych) [reply composer.json:27]: The plugin uses some classes from it, for example `Composer\Util\Filesystem`.

Comment (vjik) [reply composer.json:27]: But before that, it worked without this requirements.

Comment (devanych) [reply composer.json:27]: I removed the version, look carefully at the diff of changes.

Comment (vjik) [reply composer.json:27]: Sorry, I didn't notice

**Review by vjik — APPROVED** [review]

**Pull request merged**